python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -101,6 +105,8 @@ cpdef generate_pandas_metadata(Table table, index):
                 "'category' column dtypes are currently not "
                 + "supported by the gpu accelerated parquet writer"
             )
+        elif is_list_dtype(col):
+            types.append(col.dtype.to_arrow())
```

Comment:
This list of types is given to pyarrow to construct pandas metadata using `pa.pandas_compat.construct_metadata`. That creates the string: 
```json
{"index_columns": [], "column_indexes": [{"name": null, "field_name": null, "pandas_type": "unicode", "numpy_type": "object", "metadata": {"encoding": "UTF-8"}}], "columns": [{"name": "a", "field_name": "a", "pandas_type": "list[list[int64]]", "numpy_type": "list", "metadata": null}], "creator": {"library": "pyarrow", "version": "1.0.1"}, "pandas_version": "1.1.2"}
```
Notice the `"numpy_type": "list"` in columns metadata. This is invalid and it should be `"numpy_type": "object"`.